mobify/progressive-web-scaffold
# Remove selector utils

This PR finishes the replacement of the `selector-utils.js` file with the `reselect-immutable-helpers` NPM package. It lowers the function coverage threshold as selector-utils was well-covered with tests.

 **JIRA**: N/A

## Changes
- Switch all references to selector-utils.js to reselect-immutable-helpers
- Remove selector-utils.js

## How to test-drive this PR
- npm i
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that the app still loads with no errors.
